src/linker/Linker.Steps/ResolveFromAssemblyStep.cs

```diff
@@ -164,24 +176,31 @@ static void MarkType (LinkContext context, TypeDefinition type, RootVisibility r
 			if (type.HasNestedTypes)
 				foreach (var nested in type.NestedTypes)
 					MarkType (context, nested, rootVisibility);
+			if (type.HasInterfaces)
+				foreach (var iface in type.Interfaces) {
+					var @interface = iface.InterfaceType.Resolve ();
+					if (@interface != null && MarkType (context, @interface, rootVisibility))
+						context.Annotations.Mark (iface);
+				}
 
 			context.Tracer.Pop ();
+			return true;
 		}
 
-		void ProcessExecutable (AssemblyDefinition assembly)
+		void ProcessExecutable ()
 		{
-			Context.SetAction (assembly, AssemblyAction.Link);
+			Context.SetAction (_assembly, AssemblyAction.Link);
 
-			Tracer.Push (assembly);
+			Tracer.Push (_assembly);
 
-			Annotations.Mark (assembly.EntryPoint.DeclaringType);
+			Annotations.Mark (_assembly.EntryPoint.DeclaringType);
 
-			MarkMethod (Context, assembly.EntryPoint, MethodAction.Parse, RootVisibility.Any);
+			MarkMethod (Context, _assembly.EntryPoint, MethodAction.Parse, RootVisibility.Any);
 
 			Tracer.Pop ();
 		}
 
-		static void MarkFields (LinkContext context, Collection<FieldDefinition> fields, RootVisibility rootVisibility)
+		void MarkFields (LinkContext context, Collection<FieldDefinition> fields, RootVisibility rootVisibility)
```

Comment:
Why was this change needed?

Reply:
This was left over from a previous commit - will remove, thanks!